pkg/proxy/iptables/proxier.go

```diff
@@ -117,7 +117,7 @@ const sysctlBridgeCallIPTables = "net/bridge/bridge-nf-call-iptables"
 type serviceInfo struct {
 	*proxy.BaseServiceInfo
 	// The following fields are computed and stored for performance reasons.
-	nameString             string
+	svcptNameString        string
```

Comment:
I know Tim suggested "svcpt" in the issue, but it's a pretty unreadable name...

Maybe if you do the type renamings then you can keep the longer for ("svcPort" or "servicePort") in the type names and not need an abbreviated form in the field names.